NewForce-Cohort-9/tabloid-full-stack-debugging-dynasty
# At viewallposts #3

**Description**
Created Posts model, controller, repo/irepo, and "Posts" navbar button so you can view all posts [Trello Ticket #3](https://trello.com/c/ba2qWURq)

**Testing Instructions**
- git add and git commit whatever you're working on on your branch
- git checkout main
- git fetch --agit checkout <at-viewallposts-#3>
- If you don't see the most recent version of my files, git pull origin <at-viewallposts-#3>
- If you haven't created the Tabloid MVC database, please do so.
- Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-debugging-dynasty/blob/main/SQL/01_Tabloid_Create_DB.sql)
- Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-debugging-dynasty/blob/main/SQL/02_Tabloid_Seed_Data.sql)
- Click the execute with debugger solid green button to launch the app
- In browser, please go to Login on the nav bar and log in with this email: [foo@bar.com](mailto:foo@bar.com)
- This should redirect you back to the home page.
- You should see a "Posts" option on the navbar
- When you click on the "Posts" button it should redirect you to another page that displays the list of Posts (category will be empty)

**Checklist:**
